src/Websites/Websites/Cmdlets/DeploymentSlots/NewAzureWebAppSlot.cs

```diff
@@ -167,8 +170,8 @@ private void UpdateConfigIfNeeded(Site site)
 		public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
-            
-            CloningInfo cloningInfo = null;
+			ManagedServiceIdentity sourceIdentity = null;
+			CloningInfo cloningInfo = null;
```

Comment:
```suggestion
            ManagedServiceIdentity sourceIdentity = null;
            CloningInfo cloningInfo = null;
```
Correct indents please

Reply:
Done